README.md

```diff
@@ -130,21 +126,14 @@ py -3.12 -m pip install plasmapy
 The `3.12` may be replaced by any version of Python that is supported by
 PlasmaPy.
 
-If you have [installed Conda], then you can also install PlasmaPy into
-an activated Conda environment by running:
```

Comment:
😅 One of the steps I take when editing a document is to remove as many occurrences of "also" as I can!